app/assets/stylesheets/refinery/find_out.scss

```diff
@@ -17,31 +17,110 @@
   .banner-content {
     margin-bottom: 7.5rem;
     margin-top: 7rem;
+}
+
+/* main filter and dropdowns *******************************/
+  .find-out__filter {
```

Comment:
I've typically been using the convention 

```
.find-out {
  &__filter {
  }
}
```
In order to better organize code with SASS and BEM if you want to refactor this.

Reply:
agreed, I will implement this